Makefile

```diff
@@ -81,6 +81,10 @@ deploy: manifests kustomize
 # Generate manifests e.g. CRD, RBAC etc.
 manifests: controller-gen
 	$(CONTROLLER_GEN) $(CRD_OPTIONS) rbac:roleName=antrea-operator webhook paths="./..." output:crd:artifacts:config=config/crd/bases
+	cp config/crd/bases/operator.antrea.vmware.com_antreainstalls.yaml config/crd/operator.antrea.vmware.com_antreainstalls.yaml
```

Comment:
Should we use `kustomize build config/crd` to generate `config/crd/operator.antrea.vmware.com_antreainstalls.yaml` instead of copying directly?

Reply:
Make sense, thanks